.evergreen.yml

```diff
@@ -72,7 +72,7 @@ functions:
           export NPM_CACHE_DIR="$(pwd)/.deps/.npm"
           export NPM_TMP_DIR="$(pwd)/.deps/tmp"
 
-          export NODE_JS_VERSION="12.4.0"
+          export NODE_JS_VERSION="12.9.0"
```

Comment:
I mean, you can just make it 12.22.3 here? Doesn’t have to be the exact minimum required by the driver :)

Reply:
That's a good point! Do you know why driver needs at least 12.9.0 now? Wondering if our electron can keep up with that

Reply:
I don’t have any more context than this: https://github.com/mongodb/node-mongodb-native/pull/2864#discussion_r660037603

So I think it’s fine, at least for now